index.js

```diff
@@ -29,6 +29,15 @@ module.exports = (compiler, opts) => {
     level: options.logLevel,
     timestamp: options.logTime
   });
+  const envTarget = process.env.WHC_TARGET;
+
+  // issue #36. some alternative compiler targets still run in a server, but we
+  // don't want to be in the business of supporting them all.
+  if (!envTarget) {
+    process.env.WHC_TARGET = 'web';
+  } else if (envTarget !== 'web') {
+    log.warn(`WHC_TARGET changed to '${envTarget}'. Changing this value is allowed but is unsupported.`);
```

Comment:
absolutely should be a warn. it's warning the user that their change of that setting is unsupported. not really sure how that's not obvious.

Reply:
Okay, I don't dispute that it's warning material.

Here's why I'm asking: I'll be developing an Electron application for some time. I'm 100% certain that `electron-renderer` is equal to `web` as far as `webpack-serve` is concerned, because it's a superset of `web` feature-wise. Yet, I'll be seeing the warning every time I launch `webpack-serve`. I don't like noise in my output (that's my all my logging levels are at `warn`), and I don't want to suppress all warnings. So what do I do?

For example, Node itself has a `--no-deprecation` flag, which suppresses all deprecation warnings. Could something similar be added in this case?

Reply:
nah, that'd be feature/options bloat. it's a single line warning. if that breaks the bank for you as too much noise, then that can't be helped. you're going to be using this outside the intended environment, you're gonna have to deal with that warning.